docs/modules/integrate/pages/mongodb-connector.adoc

```diff
@@ -2,6 +2,11 @@
 
 MongoDB is a well known document-oriented NoSQL database. Hazelcast is able to connect to MongoDB to read and write data.
 
+== Requirements
+
+MongoDB should be on version at least 4.4. If you cannot connect from more than one processor (eg. if you are using
+Mongo Atlas offering with 1 client only), you have to ensure you use `forceReadTotalParallelismOne` option.
```

Comment:
2 remarks: 
I think that we should be more precise here, on our side it's `1 client only`, but from Atlas it's `JavaScript support` which is used on our side to be able to connect with more clients. Supporting JS can be also disabled in configuration on server.
2. Another one requirement - but **only for streaming** is `enabled Oplog` (again, in case of Atlas it's not supported for Serverless - but we don't have to point it out). It's important requirement in my opinion, because it forces replicaSet on mongo cluster (it won't work with out-of-the-box mongo docker image afaik).

Reply:
https://www.mongodb.com/docs/manual/core/server-side-javascript/ - it's called "Server-side JavaScript"